.devcontainer/devcontainer.json

```diff
@@ -4,7 +4,7 @@
 	"name": "Local Environment in Kubernetes",
 	// Or use a Dockerfile or Docker Compose file. More info: https://containers.dev/guide/dockerfile
 	"image": "mcr.microsoft.com/devcontainers/base:bullseye",
-	"onCreateCommand": "helm plugin install https://github.com/databus23/helm-diff --version=3.9.6",
+	"onCreateCommand": "helm plugin install https://github.com/databus23/helm-diff --version=3.9.6 && helm plugin install https://github.com/aslafy-z/helm-git --version 1.3.0", 
```

Comment:
```suggestion
        // we need helm-diff to see what has changed since the previous deploy
        // and we need helm-git to fetch a chart from a git repo
	"onCreateCommand": "helm plugin install https://github.com/databus23/helm-diff --version=3.9.6 && helm plugin install https://github.com/aslafy-z/helm-git --version 1.3.0", 
```